elements/lisk-p2p/src/p2p.ts

```diff
@@ -727,14 +735,37 @@ export class P2P extends EventEmitter {
 				}
 
 				const existingPeer = this._peerPool.getPeer(peerId);
+				const existingDuplicatePeer = this._peerPool.getDuplicatePeer(peerId);
+				// Allow connections with lower version to have incoming connection even if it has outbound
+				// tslint:disable-next-line:no-let
+				let isInboundCreated = false;
+				if (
```

Comment:
I guess we can simplify here merging this condition with the one below. In that case, we might still be able to keep the `cyclomatic-complexity`.